students/sikorski_marcin/Lesson_2_-_Python_basics.py

```diff
@@ -0,0 +1,144 @@
+#-------------------
+#Lesson_3
+#-------------------
+#Leap Year
+year = int(input("Enter a year to check: "))
+
+while year > 0:
+    if (year % 400 == 0):
+        print("LEAP")
+        break
+    else:
+        if (year%4 == 0) and (year%100 != 0):
+            print("LEAP")
+            break
+        else:
+            print("COMMON")
+            break
+
+#Chocolate bar
+firstValueOfPortion = int(input("Enter first value: "))
+secondValueOFPortion = int(input("Enter second value: "))
+numberOfSquares = int(input("Enter number of squares: "))
+
+if (numberOfSquares%firstValueOfPortion == 0 and numberOfSquares/firstValueOfPortion < secondValueOFPortion) or (numberOfSquares%secondValueOFPortion == 0 and numberOfSquares/secondValueOFPortion < firstValueOfPortion):
+    print("YES - we can divide choclate!")
+else:
+    print("NO - we can't divide chocolate")
+
+#Knight move
+firstColumn = int(input("Enter first column: "))
+firstRow = int(input("Enter first row: "))
+secondColumn = int(input("Enter second column: "))
+secondRow = int(input("Enter second row: "))
+
+row = abs(firstRow-secondRow)
+column = abs(firstColumn-secondColumn)
+
+if row == 2 and column == 1 or row == 2 and column == 1:
+    print("YES - we can move there!")
+else:
+    print("NO - we can't move there!")
+
+#Queen move
+firstColumn = int(input("Enter first column: "))
+firstRow = int(input("Enter first row: "))
+secondColumn = int(input("Enter second column: "))
+secondRow = int(input("Enter second row: "))
+
+row = abs(firstRow-secondRow)
+column = abs(firstColumn-secondColumn)
+
+
+if row == column or firstRow == secondRow or firstColumn== secondColumn:
+    print("YES - we can move the queen there!")
+else:
+    print("NO - we can't move queen there")
+
+#Bishop Moves
+firstColumn = int(input("Enter first column: "))
+firstRow = int(input("Enter first row: "))
+secondColumn = int(input("Enter second column: "))
+secondRow = int(input("Enter second row: "))
+
+row = abs(firstRow-secondRow)
+column = abs(firstColumn-secondColumn)
+
+if row == column:
+    print("YES - we can move bishop there!")
+else:
+    print("NO - we can't move bishop there!")
+
+#---------------------------
+#Lesson 4
+#---------------------------
+
+#Lost cards
+numberOfCards = int(input("Enter number of cards: "))
+sumOfNumberOfCards = (((1 + numberOfCards)/2) * (numberOfCards))
+
+sum = 0
+while numberOfCards != 1:
```

Comment:
Ponieważ znana jest liczba powtórzeń, odpowiadająca liczbie kart, to powinno się zastosować pętlę `for`, wykorzystaj funkcję `range`